src/common/io.hpp

```diff
@@ -297,9 +297,9 @@ class BufferedReader : virtual public Reader {
 	bool rewind_consumed_ {false};
 	bool stop_done_ {false};
 
-protected:
```

Comment:
This was meant for testing - see the CI failure

Reply:
Right.